code/datums/achievements/misc_scores.dm

```diff
@@ -37,14 +37,16 @@
 
 /datum/award/score/progress/fish/proc/validate_early_joiners(datum/source)
 	for(var/client/client as anything in GLOB.clients)
-		var/datum/achievement_data/holder = client.player_details.achievements
+		var/datum/achievement_data/holder = client.persistent_client.achievements
```

Comment:
should prolly use a `?.` for the client access

Reply:
client is guaranteed here